xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/Registration.xml

```diff
@@ -487,6 +505,16 @@
   ##
 #end## createUser Macro
 {{/velocity}}</content>
+  <attachment>
+    <filename>undraw_done_re_oak4.svg</filename>
```

Comment:
Small detail but would be better to have a name more understandable.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3155/commits/f7671c16c40b50364cf3516ca1c347cb7a1d96cf :+1: